src/test/java/no/unit/alma/user/AlmaUsersRequestsTest.java

```diff
@@ -0,0 +1,140 @@
+        UserRequest resultUserRequest = almaUsersRequestService.getUserRequest(TEST_USER_ID, TEST_REQUEST_ID);
+        assertEquals(TEST_AUTHOR, resultUserRequest.getAuthor());
+
+    }
+
+    @Test
+    void testGetUserRequests() {
+        when(webTarget.path(anyString())).thenReturn(webTarget);
+        when(webTarget.queryParam(anyString(), any())).thenReturn(webTarget);
+        when(webTarget.request()).thenReturn(builder);
+        when(builder.accept(anyString())).thenReturn(builder);
+        when(builder.buildGet()).thenReturn(invocation);
+
+        UserRequests testUserRequests = new UserRequests();
+        testUserRequests.setTotalRecordCount(TOTAL_RECORD_COUNT);
+        when(invocation.invoke(UserRequests.class)).thenReturn(testUserRequests);
+
+        AlmaUsersRequests almaUsersRequestService = new AlmaUsersService(mockAlmaApiClient);
+        UserRequests resultUserRequests = almaUsersRequestService.getUserRequests(TEST_USER_ID);
+        assertEquals(TOTAL_RECORD_COUNT, resultUserRequests.getTotalRecordCount());
+    }
+
+    @Test
+    void testDeleteUserRequest() {
+        when(webTarget.path(anyString())).thenReturn(webTarget);
+        when(webTarget.request()).thenReturn(builder);
+        when(builder.accept(anyString())).thenReturn(builder);
+        when(builder.buildDelete()).thenReturn(invocation);
+        when(invocation.invoke()).thenReturn(response);
+
+        AlmaUsersRequests almaUsersRequestService = new AlmaUsersService(mockAlmaApiClient);
+        almaUsersRequestService.deleteUserRequest(TEST_USER_ID, TEST_REQUEST_ID);
+    }
+
+    @Test
+    void testPostUserRequest() {
+        when(webTarget.path(anyString())).thenReturn(webTarget);
+        when(webTarget.queryParam(anyString(), any())).thenReturn(webTarget);
+        when(webTarget.request()).thenReturn(builder);
+        when(builder.accept(anyString())).thenReturn(builder);
+        when(builder.buildPost(any())).thenReturn(invocation);
+
+        UserRequest testUserRequest = new UserRequest();
+        testUserRequest.setAuthor(TEST_AUTHOR);
+        when(invocation.invoke(UserRequest.class)).thenReturn(testUserRequest);
+
+        AlmaUsersRequests almaUsersRequestService = new AlmaUsersService(mockAlmaApiClient);
+        UserRequest resultUserRequest =
+                almaUsersRequestService.postUserRequest(TEST_USER_ID, TEST_REQUEST_ID, testUserRequest);
+        assertEquals(TEST_AUTHOR, resultUserRequest.getAuthor());
+
+        resultUserRequest =
+                almaUsersRequestService.postUserRequestItem(TEST_USER_ID, TEST_ITEM_ID, testUserRequest);
+        assertEquals(TEST_AUTHOR, resultUserRequest.getAuthor());
+    }
+
+    // UserRequest postUserRequest(String userIdentifer, String recordIdentifer,
+    // UserRequest userRequest);
+    //
+    // UserRequest postUserRequestItem(String userIdentifer, String itemId,
+    // UserRequest userRequest);
+    //
```

Comment:
Hvorfor er det commentert og ikke slettet? Trenger vi det på en senere tidspunkt

Reply:
ups, brukte det for å ha oversikt, sletter før jeg merger